feature/bgp/policybase/otg_tests/chained_policies/README.md

```diff
@@ -0,0 +1,266 @@
+# RT-1.29: BGP chained import/export policy attachment
+
+## Summary
+
+- A list of policies to be attached to a neighbor's import-policy
+- A list of policies to be attached to a neighbor's export-policy
+- Applicable to both IPv4 and IPv6 BGP neighbors
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed
+
+## Procedure
+
```

Comment:
In the future we may change the style guide to require this.  This more accurately represents the way we apply configuration and sometimes reveals issues with DUT behavior.

```suggestion
### Applying configuration

For each section of configuration below, prepare a gnmi.SetBatch  with all the configuration items appended to one SetBatch.  Then apply the configuration to the DUT in one gnmi.Set using the `replace` option.

```